distribution/pom.xml

```diff
@@ -333,6 +333,8 @@
                                         <argument>org.apache.druid.extensions.contrib:druid-time-min-max</argument>
                                         <argument>-c</argument>
                                         <argument>org.apache.druid.extensions.contrib:druid-virtual-columns</argument>
+                                        <argument>-c</argument>
+                                        <argument>io.druid.extensions.contrib:druid-tdigestsketch</argument>
```

Comment:
This needs to be `org.apache.druid.extensions.contrib` now

Reply:
Ah! I initially built this against 0.12.2 since that is what we use at my day job. Fixed.